documents/knowledge/elements.yaml

```diff
@@ -1651,12 +1654,16 @@
   tests:
     - '["abcdef",2] : ["ab","cd","ef"]'
     - "[[1,2,3,4,5,6],3] : [[1,2,3],[4,5,6]]"
+    - '["abcdefghi",[2,3,4]] : ["ab","cde","fghi"]'
+    - "[[1,2,3,4,5], [2,3] ] : [[1,2],[3,4,5]]"
 
 - element: "ẋ"
   name: Repeat
   description: Repeat a value several times
   arity: "*"
   overloads:
+    str-num: a * b
+    num-str: b * a
     any-num: Repeat a b times
```

Comment:
Nope, `str-num` takes precedence because `'abc' * 3` is `'abcabcabc'`, but if we treated it like `any-num`, then it would be `['abc', 'abc', 'abc']`. I'm not convinced the str-num overload is needed at all, since we have `*` anyway, but /shrug.

Reply:
Ah, ok - I thought "repeat a b times" meant `a * b` as well, but it's actually `[a] * b`. In that case, this string behaviour is now annoyingly inconsistent with lists - can I suggest we don't change this, given we have `*` already?